components/net/Cargo.toml

```diff
@@ -34,6 +34,7 @@ plugins = {path = "../plugins"}
 profile_traits = {path = "../profile_traits"}
 rand = "0.3"
 rustc-serialize = "0.3"
+script = {path = "../script"}
```

Comment:
We need to find a way not to do this; it's useful for net to be able to build without depending on script.